spec/spec_helper.rb

```diff
@@ -33,13 +33,13 @@
 
 # Required for running background jobs on demand (deterministically)
 ActiveJob::Base.queue_adapter = :test
-# Required for serializing objects in similar to production environments
+# Required for serializing objects similar to production environments
 GlobalID.app = 'meilisearch-test'
 
 OLD_RAILS = Gem.loaded_specs['rails'].version < Gem::Version.new('4.0')
 NEW_RAILS = Gem.loaded_specs['rails'].version >= Gem::Version.new('6.0')
 
-Dir["#{File.dirname(__FILE__)}/support/**/*.rb"].sort.each { |file| require file }
+Dir["#{File.dirname(__FILE__)}/support/*.rb"].sort.each { |file| require file }
```

Comment:
Only auto load the first level of support files. The other ones are intended to be loaded where needed to have fewer mystery guests.